manifest.yaml

```diff
@@ -1,52 +1,50 @@
 version: "2"
 
-# TODO: Replace with the name of your Airdrop connector
-name: Template Connector
-# TODO: Replace with the description for your Airdrop connector
-description: Template Connector for importing dummy data into DevRev
+# When filling out the manifest.yaml, please follow these instructions:
+# 1. Replace all ALL_CAPS_TEMPLATES with the appropriate values for your connector.
+# 2. Name the <VARIABLES> and make sure the references match.
+# 3. Keep all lower case variables as they are.
```

Comment:
This manifest is only so that we can run snap-in commands on the template itself, so instructions like these are not going to be seen by anyone. We should actually remove it alltogether.